test/system/src/main/java/io/pravega/test/system/framework/services/kubernetes/AbstractService.java

```diff
@@ -261,7 +268,7 @@ private V1beta1RoleBinding getPravegaOperatorRoleBinding() {
 
     private V1Deployment getPravegaOperatorDeployment() {
         V1Container container = new V1ContainerBuilder().withName(PRAVEGA_OPERATOR)
-                                                        .withImage("pravega/pravega-operator:" + PRAVEGA_OPERATOR_VERSION)
+                                                        .withImage(PRAVEGA_OPERATOR_IMAGE)
```

Comment:
are we planning to have snapshot builds for pravega-operator?

Reply:
Using custom operator images is needed to test out preliminary changes and also on environments where no access to Docker Hub is possible and images are host in private registries.